src/main/java/uk/gov/companieshouse/company/profile/controller/CompanyProfileController.java

```diff
@@ -27,12 +29,13 @@ public CompanyProfileController(CompanyProfileService companyProfileService) {
      * @return company profile api
      */
     @GetMapping("/company/{company_number}")
-    public ResponseEntity<CompanyProfile> getCompanyProfile(
+    public ResponseEntity<Data> getCompanyProfile(
             @PathVariable("company_number") String companyNumber) {
         return companyProfileService.get(companyNumber)
                 .map(companyProfileDao ->
                         new ResponseEntity<>(
-                                new CompanyProfile().data(companyProfileDao.companyProfile),
+                                new CompanyProfile()
+                                        .data(companyProfileDao.companyProfile).getData(),
```

Comment:
this will work, but we can simply do new ResponseEntity<>(companyProfileDao.companyProfile.data) to get same response